diff --git a/es/Trigger/index.js b/es/Trigger/index.js
--- a/es/Trigger/index.js
+++ b/es/Trigger/index.js
@@ -332,7 +332,7 @@ var Trigger = /** @class */ (function (_super) {
             var content = _this.triggerRef.current;
             var child = _this.getRootElement();
             // offsetParent=null when display:none or position: fixed
-            if (!child.offsetParent && !child.getClientRects().length) {
+            if (!child || (!child.offsetParent && !child.getClientRects().length)) {
                 return _this.state.popupStyle;
             }
             var mergedProps = _this.getMergedProps();
diff --git a/lib/Trigger/index.js b/lib/Trigger/index.js
--- a/lib/Trigger/index.js
+++ b/lib/Trigger/index.js
@@ -357,7 +357,7 @@ var Trigger = /** @class */ (function (_super) {
             var content = _this.triggerRef.current;
             var child = _this.getRootElement();
             // offsetParent=null when display:none or position: fixed
-            if (!child.offsetParent && !child.getClientRects().length) {
+            if (!child || (!child.offsetParent && !child.getClientRects().length)) {
                 return _this.state.popupStyle;
             }
             var mergedProps = _this.getMergedProps();
